.travis.yml

```diff
@@ -20,26 +20,12 @@ after_success:
 - bash <(curl -s https://codecov.io/bash)
 - bash kubernetes/travis/deploy.sh
 - codecov
-before_deploy:
```

Comment:
The key is required to clone and publish docs using ssh. I dont think it'll be needed anywhere else.